app/src/main/java/org/asteroidos/sync/MainActivity.java

```diff
@@ -281,82 +306,48 @@ public void onLocationSettingsClicked() {
     }
 
     private void handleSetLocalName(String name) {
-        if(mDetailFragment != null)
+        if (mDetailFragment != null)
             mDetailFragment.setLocalName(name);
     }
 
     private void handleSetStatus(int status) {
-        if(mDetailFragment != null) {
+        if (mDetailFragment != null) {
             mDetailFragment.setStatus(status);
-            if(status == SynchronizationService.STATUS_CONNECTED) {
-                try {
-                    Message batteryMsg = Message.obtain(null, SynchronizationService.MSG_REQUEST_BATTERY_LIFE);
-                    batteryMsg.replyTo = mDeviceDetailMessenger;
-                    mSyncServiceMessenger.send(batteryMsg);
-                } catch (RemoteException ignored) {}
-            }
             mStatus = status;
         }
+        if (status == SynchronizationService.STATUS_CONNECTED)
```

Comment:
nit: I'd prefer if we would keep {} around the try{}catch[} block because it spans multi line